docs/release-notes/NuGet-6.8.md

```diff
@@ -0,0 +1,172 @@
+
+* Improve hashing and equality allocations/performance - [#12746](https://github.com/NuGet/Home/issues/12746)
+
+* NuGetAudit severity bugs - [#12743](https://github.com/NuGet/Home/issues/12743)
+
+* Lock contention thread pool issues caused by LoadSettings not passing settingsLoadingContext to LoadSettingsForSpecificConfigs - [#12737](https://github.com/NuGet/Home/issues/12737)
+
+* NuGetAuditMode all warns about package versions that were upgraded (rejected) - [#12730](https://github.com/NuGet/Home/issues/12730)
+
+* An error “unable to find metadata of PackageName.1.0.0” occurs when installing package with “packages.config” format - [#12723](https://github.com/NuGet/Home/issues/12723)
+
+* WalkTreeRejectNodesOfRejectedNodes constantly triggering resizes of its tracker collection - [#12719](https://github.com/NuGet/Home/issues/12719)
+
+* Reduce RuntimeGraph allocations as it's immutable - [#12717](https://github.com/NuGet/Home/issues/12717)
+
+* Heavy allocations in NuGet.Commands.RestoreRunner.ExecuteAndCommitAsync|nuget.packaging.dll!NuGet.RuntimeModel.RuntimeDescription - [#12714](https://github.com/NuGet/Home/issues/12714)
+
+* Heavy allocations in NuGet.Commands.RestoreRunner.ExecuteAndCommitAsync|nuget.versioning.dll!NuGet.Versioning.VersionFormatter.Format - [#12707](https://github.com/NuGet/Home/issues/12707)
+
+* Remove allocations from PackageSource.Source setter - [#12692](https://github.com/NuGet/Home/issues/12692)
+
+* ContentItemCollection.FindBestItemGroup boxing enumerator - [#12689](https://github.com/NuGet/Home/issues/12689)
+
+* FrameworkNameProvider.GetVersionString boxing enumerator - [#12685](https://github.com/NuGet/Home/issues/12685)
+
+* NuGet.Client allocates many instances of comparers - [#12680](https://github.com/NuGet/Home/issues/12680)
+
+* GetContentFileFolderRelativeToFramework allocates too much - [#12668](https://github.com/NuGet/Home/issues/12668)
+
+* Deprecated info will flash for less than one second in the right penal when clicking package “Microsoft.Net.Http” with a non-deprecated version in the package list - [#12661](https://github.com/NuGet/Home/issues/12661)
+
+* CreateGraphNode has a high number of allocations - [#12641](https://github.com/NuGet/Home/issues/12641)
+
+* The vulnerable label doesn’t show in the “version” dropdown box of “Browse” tab when searching for vulnerable packages  - [#12623](https://github.com/NuGet/Home/issues/12623)
+
+* NuGet.Commands.LockFileBuilder  KeyNotFoundException Exception - [#12464](https://github.com/NuGet/Home/issues/12464)
+
+* A PackageDownload without a version causes a NullReferenceException - [#12212](https://github.com/NuGet/Home/issues/12212)
+
+* [Bug]: View License dialog does not display license content - [#12060](https://github.com/NuGet/Home/issues/12060)
+
+* Package Source Mapping UI - Package Installation/update through PM UI - [#11366](https://github.com/NuGet/Home/issues/11366)

```

Comment:
The issue title is kind of generic, since it was sort of a tracking issue for the work overall. The PR title I used is more specific that this is about automatically creating mappings (we already introduced status/configure in 17.5 and 17.7). Should I rename the issue or just put the PR title here?

Reply:
Feel free to make a suggestion here. 

In retrospect, I feel like that should've been a more prominent item as it's new feature support. 
Should we add it in the What's New section, or is it ok if it's here?